### PR TITLE
Fixing ternary operator in ImageRuntime (v2)

### PR DESCRIPTION
**Pull request by Boorj**
again, as in #7805

### CODE REVIEW

File: src/Twig/Runtime/ImageRuntime.php [hunk 1]
```diff
@@ -345,9 +345,12 @@ private function normalizeFileName($fileName)
         if (!is_array($fileName)) {
             return $fileName;
         }
-
-        return isset($fileName['filename']) ? $fileName['filename']
-            : isset($fileName['file']) ? $fileName['file'] : ''
-        ;
+        if (isset($fileName['filename']) {
```
Comment (bobdenotter) [inline src/Twig/Runtime/ImageRuntime.php]:
```suggestion
        if (isset($fileName['filename'])) {
```

File: src/Twig/Runtime/ImageRuntime.php [hunk 2]
```diff
@@ -345,9 +345,12 @@ private function normalizeFileName($fileName)
...
+            return $fileName['filename'];
+        }
+        if (isset($fileName['file']) {
```
Comment (bobdenotter) [inline src/Twig/Runtime/ImageRuntime.php]:
```suggestion
        if (isset($fileName['file'])) {
```

^ Two typos

**Review by bobdenotter — CHANGES_REQUESTED** [review]

**Comment by Boorj** [comment]
@bobdenotter isn't it ok now?

**Review by bobdenotter — APPROVED** [review]
@Boorj  Sorry, I missed that last update. Yes, looks good now! 👍

**Pull request merged**